### PR TITLE
fix  hard code and make/zh/docs/setup/release/version-skew-policy.md up-to-date

### PR DESCRIPTION
**Pull request by howieyuen**
fix: https://github.com/kubernetes/website/issues/24462

### CODE REVIEW

File: content/zh/docs/setup/release/version-skew-policy.md [hunk 1]
```diff
@@ -87,13 +84,13 @@ Example:
...
-* `kubelet` is supported at **1.13**, **1.12**, and **1.11**
+* `kube-apiserver` is at **{{< skew latestVersion >}}**
+* `kubelet` is supported at **{{< skew latestVersion >}}**, **{{< skew prevMinorVersion >}}**, and **{{< skew oldestMinorVersion >}}**
 -->
 例如：
 
-* `kube-apiserver` 版本号如果是 **1.13**
-* `kubelet` 只能是 **1.13** 、 **1.12** 和 **1.11**
+* `kube-apiserver` 版本号如果是 **{{< skew latestVersion >}}**
+* `kubelet` 只能是 **{{< skew latestVersion >}}**、**{{< skew prevMinorVersion >}}** 和 **{{< skew oldestMinorVersion >}}**
```
Comment (tengqm) [inline content/zh/docs/setup/release/version-skew-policy.md:93]:
```suggestion
* 受支持的 `kubelet` 版本将包括 **{{< skew latestVersion >}}**、**{{< skew prevMinorVersion >}}** 和 **{{< skew oldestMinorVersion >}}**
```

File: content/zh/docs/setup/release/version-skew-policy.md [hunk 2]
```diff
@@ -105,13 +102,14 @@ If version skew exists between `kube-apiserver` instances in an HA cluster, this
...
-* `kube-apiserver` instances are at **1.13** and **1.12**
-* `kubelet` is supported at **1.12**, and **1.11** (**1.13** is not supported because that would be newer than the `kube-apiserver` instance at version **1.12**)
+* `kube-apiserver` instances are at **{{< skew latestVersion >}}** and **{{< skew prevMinorVersion >}}**
+* `kubelet` is supported at **{{< skew prevMinorVersion >}}**, and **{{< skew oldestMinorVersion >}}** (**{{< skew latestVersion >}}** is not supported because that would be newer than the `kube-apiserver` instance at version **{{< skew prevMinorVersion >}}**)
 -->
 例如：
 
-* 如果 `kube-apiserver` 的多个实例同时存在 **1.13** 和 **1.12**
-* `kubelet` 只能是 **1.12** 或 **1.11**（**1.13** 不再支持，因为它比**1.12**版本的 `kube-apiserver` 更新）
+* 如果 `kube-apiserver` 的多个实例同时存在 **{{< skew latestVersion >}}** 和 **{{< skew prevMinorVersion >}}**
```
Comment (tengqm) [inline content/zh/docs/setup/release/version-skew-policy.md:110]:
```suggestion
* 如果 `kube-apiserver` 实例同时存在 **{{< skew latestVersion >}}** 和 **{{< skew prevMinorVersion >}}**
```

File: content/zh/docs/setup/release/version-skew-policy.md [hunk 3]
```diff
@@ -105,13 +102,14 @@ If version skew exists between `kube-apiserver` instances in an HA cluster, this
...
+* `kubelet` 只能是 **{{< skew prevMinorVersion >}}** 或 **{{< skew oldestMinorVersion >}}**
```
Comment (tengqm) [inline content/zh/docs/setup/release/version-skew-policy.md:111]:
```suggestion
* `kubelet` 的受支持版本将是 **{{< skew prevMinorVersion >}}** 和 **{{< skew oldestMinorVersion >}}**；
```

File: content/zh/docs/setup/release/version-skew-policy.md [hunk 4]
```diff
@@ -212,36 +211,36 @@ Pre-requisites:
...
-  * The webhooks are able to handle any new versions of REST resources that will be sent to them, and any new fields added to existing versions in **1.(n+1)**
+  * `ValidatingWebhookConfiguration` and `MutatingWebhookConfiguration` objects are updated to include any new versions of REST resources added in **{{< skew latestVersion >}}** (or use the [`matchPolicy: Equivalent` option](/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-matchpolicy) available in v1.15+)
+  * The webhooks are able to handle any new versions of REST resources that will be sent to them, and any new fields added to existing versions in **{{< skew latestVersion >}}**
 -->
-* 单实例集群时，`kube-apiserver` 实例版本号须是 **1.n**
-* HA 集群时，所有的 `kube-apiserver` 实例版本号必须是 **1.n** 或 **1.(n+1)**（确保满足最新和最旧的实例小版本号相差不大于1）
-* `kube-controller-manager`、`kube-scheduler` 和 `cloud-controller-manager` 版本号必须为 **1.n**（确保不高于 API server 的版本，且版本号相差不大于1）
-* `kubelet` 实例版本号必须是 **1.n** 或 **1.(n-1)**（确保版本号不高于 API server，且版本号相差不大于2）
+* 单实例集群时，`kube-apiserver` 实例版本号须是 **{{< skew prevMinorVersion >}}**
```
Comment (tengqm) [inline content/zh/docs/setup/release/version-skew-policy.md:222]:
```suggestion
* 单实例集群中，`kube-apiserver` 实例版本号须是 **{{< skew prevMinorVersion >}}**
```

File: content/zh/docs/setup/release/version-skew-policy.md [hunk 5]
```diff
@@ -212,36 +211,36 @@ Pre-requisites:
...
+* HA 集群时，所有的 `kube-apiserver` 实例版本号必须是 **{{< skew prevMinorVersion >}}** 或 **{{< skew latestVersion >}}**（确保满足最新和最旧的实例小版本号相差不大于1）
```
Comment (tengqm) [inline content/zh/docs/setup/release/version-skew-policy.md:223]:
```suggestion
* 高可用（HA）集群中，所有的 `kube-apiserver` 实例版本号必须是 **{{< skew prevMinorVersion >}}** 或 **{{< skew latestVersion >}}**（确保满足最新和最旧的实例小版本号相差不大于1）
```

**Review by tengqm — COMMENTED** [review]
LGTM other than some nits.

**Review by irvifa — COMMENTED** [review]
/approve

**Pull request merged**